src/amo/components/AddonReviewManager/index.js

```diff
@@ -0,0 +1,144 @@
+/* @flow */
+import makeClassName from 'classnames';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import {
+  SAVED_RATING,
+  STARTED_SAVE_RATING,
+  STARTED_SAVE_REVIEW,
+  updateAddonReview,
+} from 'amo/actions/reviews';
+import { withFixedErrorHandler } from 'core/errorHandler';
+import translate from 'core/i18n/translate';
+import { sanitizeHTML } from 'core/utils';
+import DismissibleTextForm from 'ui/components/DismissibleTextForm';
+import Rating from 'ui/components/Rating';
+import type { AppState } from 'amo/store';
+import type { DispatchFunc } from 'core/types/redux';
+import type { ErrorHandlerType } from 'core/errorHandler';
+import type { FlashMessageType, UserReviewType } from 'amo/actions/reviews';
+import type { I18nType } from 'core/types/i18n';
+import type { OnSubmitParams } from 'ui/components/DismissibleTextForm';
+
+import './styles.scss';
+
+type Props = {|
+  review: UserReviewType,
+|};
+
+type InternalProps = {|
+  ...Props,
+  dispatch: DispatchFunc,
+  errorHandler: ErrorHandlerType,
+  i18n: I18nType,
+  flashMessage?: FlashMessageType | void,
+|};
+
+export class AddonReviewManagerBase extends React.Component<InternalProps> {
+  onSubmitRating = (rating: number) => {
+    const { errorHandler, dispatch, review } = this.props;
+
+    dispatch(
+      updateAddonReview({
+        rating,
+        errorHandlerId: errorHandler.id,
+        reviewId: review.id,
+      }),
+    );
+  };
+
+  onSubmitReview = ({ text }: OnSubmitParams) => {
+    const { errorHandler, dispatch, review } = this.props;
+
+    dispatch(
+      updateAddonReview({
+        body: text,
+        errorHandlerId: errorHandler.id,
+        reviewId: review.id,
+      }),
+    );
+  };
+
+  render() {
+    const { errorHandler, i18n, review, flashMessage } = this.props;
+
+    const reviewGuide = i18n.sprintf(
+      i18n.gettext(
+        'Please follow our %(linkStart)sreview guidelines%(linkEnd)s.',
+      ),
+      {
+        linkStart: '<a href="/review_guide">',
```

Comment:
Should this open in a new tab?

Reply:
I guess it would be nice but I can't figure out how to do it. When I add `target=_blank` it gets stripped out by `sanitizeHTML()`.

Reply:
If you add both target and rel, does that get stripped out too?

Reply:
I just tried `rel="noopener noreferrer"` and target is still stripped.

Upon digging, it looks like we could do it with a [hook](https://github.com/cure53/DOMPurify/blob/master/demos/hooks-target-blank-demo.html#L31) but this might be a bit much to fit into the patch. I see that Discopane does something similar but that changes all links.

Reply:
> I just tried rel="noopener noreferrer" and target is still stripped.

😞 

> Upon digging, it looks like we could do it with a hook but this might be a bit much to fit into the patch. I see that Discopane does something similar but that changes all links.

Yeah I remember having done that in https://github.com/mozilla/addons-frontend/issues/3554. Before, we had target/rel attributes on links everywhere.

We could maybe allow `target` and `rel` (`ALLOWED_ATTR`) or write a different hook.

Reply:
Allowing any kind of target value seems risky so I'd rather not do that.

Reply:
could this be a Link tag (if we are not using target=_blank)?

Reply:
I mean if we are linking internally I guess I thought the Link tag was usually better to keep as SPA?